wp-content/themes/citylimits/style.css

```diff
@@ -1819,3 +1822,4 @@ h1.hidden-page-title {
 .archive-background .rss-subscribe-link {
   display: none;
 }
+>>>>>>> master
```

Comment:
This is the remnants of a merge conflict and should be removed.

Reply:
fixed